evap/staff/staff_mode.py

```diff
@@ -52,7 +53,9 @@ def is_in_staff_mode(request):
 
 
 def update_staff_mode(request):
-    assert request.user.has_staff_permission
+    if not request.user.has_staff_permission:
+        exit_staff_mode(request)
+        return
```

Comment:
@richardebeling @Kakadus Should we log something here?

Reply:
I'm fine with both adding and not adding logging.

Reply:
If you both don't care, I would prefer if we added a small info-level log like "user {} tried to update staff mode session without permission" @ybrnr